pkg/skaffold/errors/deploy_problems.go

```diff
@@ -29,20 +28,20 @@ var (
 	isMinikube = cluster.GetClient().IsMinikube
 )
 
-func suggestDeployFailedAction(runCtx runcontext.RunContext) []*proto.Suggestion {
-	if isMinikube(runCtx.KubeContext) {
+func suggestDeployFailedAction(cfg Config) []*proto.Suggestion {
+	if isMinikube(cfg.GetKubeContext()) {
```

Comment:
I'm fine with keeping this for now, but IMHO the config should have the _cluster_, which we can query if it's local, or specifically minikube or kind, etc.

And then we can have the clusters provide a status-check command.  The generic one is `kubectl describe nodes` and `kubectl get events`.

Reply:
i think `config.ClusterType()` could be a good addition so we don't have to repeat the logic many times.